proxy/README.md

```diff
@@ -0,0 +1,16 @@
+# MIRACL Proxy
+
+We have a proxy implementation using the `goproxy` library. We need this in order to be able to tamper with requests and responses made between the samples and our platform. We also use it to test if the OIDC libraries which we use in all samples work through a proxy.
+
+`HOST` - Host to listen on. The default is "0.0.0.0".
+
+`PORT` - Port of the listening host. The default is "8080".
+
+`VERBOSE`- Should every proxied request be logged to stdout. The default value is false.
```

Comment:
maybe: `VERBOSE`- Specify should every proxied...

Reply:
Actually I prefer it as it is.

Reply:
My rule of thumb is answering the question "What will happen if this is set? It will ....". It should be in present-tense, imperative-style - log, not logs, will log or logged.
My suggestion:
> Log all requests to stdout.  The default value is false.